# Python_Project2_Draft1
How do the functional domains and predicted protein structures of human tafazzin compare with other orthologs and related phospholipid acyltransferases?

To run the code, you will need:

1) The Jupyter Notebook
2) The fasta file containing the TAZ and related acyltransferase amino acid sequences
3) The pdb files for each TAZ ortholog and related acyltransferase

**Information on where the fasta file is found**: I compiled amino acid sequences of different tafazzin orthologs from NCBI onto the same fasta file. Using the dropdown menu on the search bar, I set the category to gene and typed in "tafazzin" in the search bar. Once navigating to the page of the tafazzin protein I wanted (for example, human tafazzin is the first entry), I scrolled down to the "NCBI Reference Sequences (RefSeq)" header and clicked on the link for tafazzin isoform 1 under the "mRNA and Protein(s)" subheader. This will show information on the protein. I clicked on "FASTA" to view it in FASTA format, then clicked "send to" --> "file" to download it as a FASTA file.

**Information on where the pdb files are found**: I went onto UniProt and typed in "tafazzin" in the search bar. Then, I clicked on the entry that I wanted to view or download the pdb file from. I scrolled down to the "Structure" heading and downloaded the pdb file (most were from AlphaFold).
